homeassistant/components/anthemav/media_player.py

```diff
@@ -33,14 +39,21 @@ async def async_setup_entry(
     name = config_entry.title
     mac_address = config_entry.data[CONF_MAC]
     model = config_entry.data[CONF_MODEL]
+    volume_step = config_entry.data.get(CONF_VOLUME_STEP, DEFAULT_VOLUME_STEP)
```

Comment:
Only tiny suggestion is asserting this is greater than 0. You cover off the negative numbers in vol up/down.

Reply:
(okay, well, the min/max also covers this a bit; so only step of 0)

Reply:
Ah, TIL about voluptuous.All, I'm adding an extra validation to ensure this is >0 and never =0.